prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,40 @@
+const number1 = 23;
+const number2 = 3;
+product = number1 * number2;
```

Comment:
Don't forget to include a variable initializer! If you forget it, the browser will add a `var` for you, but we want to utilize the functionality that `const` and `let` give us